governance/README.md

```diff
@@ -0,0 +1,22 @@
+# Aave Governance Events
+
+## Description
+
+This bot monitors transactions for Aave governance events.
+
+## Supported Chains
+
+- Ethereum Mainnet
+
+## Alerts
+
+<!-- -->
+- AE-AAVE-GOVERNANCE
+  - Fired when a transaction log contains an event that matches a list of Aave governance events
+  - Severity is always set to "low"
+  - Type is always set to "suspicious"
+  - Metadata field contains Aave contract name and address, event name, and transaction hash
+
+## Test Data
+
+To run all the tests for this bot, use the following command: `npm run test`
```

Comment:
It would be good to provide several transaction hashes that this bot can be tested with.  For example, the following transaction should cause the bot to create a finding for a "VoteEmitted" event:
`0x8295f70196826dc5e1390f97330e537b5ff0183bd5c22c9a81f000cda15e8e45`

Run with:
`npm run tx 0x8295f70196826dc5e1390f97330e537b5ff0183bd5c22c9a81f000cda15e8e45`